report/renderer/category-renderer.js

```diff
@@ -144,6 +144,35 @@ export class CategoryRenderer {
     return auditEl;
   }
 
+  /**
+   * Inject the final screenshot next to the score gauce of the first category (likely Performance)
+   * @param {HTMLElement} categoriesEl
+   * @param {LH.ReportResult['audits']} audits
+   * @param {Element?} scoreScaleEl
+   */
+  injectFinalScreenshot(categoriesEl, audits, scoreScaleEl) {
+    // TODO: Use full-page-screenshot instead as that's always gathered, regardless of category filter
+    const audit = audits['final-screenshot'];
+    if (!audit || audit.scoreDisplayMode === 'error') return null;
+    const details = audit.details;
+    if (!details || details.type !== 'screenshot') return null;
+    const finalScreenshotDataUri = details.data;
```

Comment:
```suggestion
    if (!audit.details || audit.details.type !== 'screenshot') return null;

    const finalScreenshotDataUri = audit.details.data;
```

readability improvement